src/routes/todoRoutes.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from "express";
+import { TodoController } from "../controllers/todoController";
+
+
+export function createTodoRouter(todoController: TodoController): Router{
+    const router = Router();
+    router.get("/", todoController.getTodos);
+    router.post("/", todoController.createNewTodo); //need to reed from req
+    //something that i have learned here is that in express routes are matched in the order they're defined :)
+    // so in the start i do the regular delete first and then every time i do delete all (/all) i got my message
+    // that this id isnt in the list.
```

Comment:
nice, but shouldn't be part of the code :)
if you think it is important is should be written as a "Comment", something like
"Note that we must place the "all" call before ":id" to avoid..."